database/migrations/2016_02_27_103851_create_osu_slack_users_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateOsuSlackUsersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('osu_slack_users', function (Blueprint $table) {
+            $table->string('slack_id', 50);
+            $table->integer('user_id');
```

Comment:
I'm going to modify the column from my end, so let's just assume it's in the correct state from the beginning (should be fine to use mediuminteger type in the create statements).